doc/examples/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -99,23 +112,24 @@ Save this file in the config folder in the robot's moveit_config package with na
 
 * *shadow_threshold*: The minimum brightness of the shadow map below an entity for its dynamic shadow to be visible
 
-* *padding_offset*: The size of the padding (in cm).
+* *padding_scale*: Should always be >= 1.0. Scale up collision shapes in the scene before excluding them from the octomap.
 
-* *padding_scale*: The scale of the padding.
+* *padding_offset*: Absolute padding (in m) around scaled collision shapes when excluding them from the octomap.
 
 * *filtered_cloud_topic*: The topic on which the filtered cloud will be published (mainly for debugging). The filtering cloud is the resultant cloud after self-filtering has been performed.
 
+* *ns*: An optional namespace for the advertised topics. Required for multiple sensors of the same type.
 
 Update the launch file
 ++++++++++++++++++++++
 
 Add the YAML file to the launch script
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-You will now need to update the *sensor_manager.launch* file in the "launch" directory of your panda_moveit_config directory with this sensor information (this file is auto-generated by the Setup Assistant but is empty). You will need to add the following line into that file to configure the set of sensor sources for MoveIt to use: ::
+You will now need to create a *sensor_manager.launch* file in the "launch" directory of your panda_moveit_config directory (e.g. `on github <https://github.com/ros-planning/panda_moveit_config/blob/rolling-devel/launch/sensor_manager.launch.xml>`_) with this sensor information. You will need to add the following line into that file to configure the set of sensor sources for MoveIt to use: ::
 
- <rosparam command="load" file="$(find panda_moveit_config)/config/sensors_kinect_pointcloud.yaml" />
+ <param from="$(find panda_moveit_config)/config/sensors_3d.yaml" />
 
-If you are using depthmap change the name of the yaml file to ``sensors_kinect_depthmap.yaml``.
+If you are using depthmap change the name of the yaml file to ``sensors_3d.yaml``.
 Note that you will need to input the path to the right file you have created above.
 
 Octomap Configuration
```

Comment:
Perhaps this section needs to be refactored for ROS2 as well? In your fork of the [panda moveit config](https://github.com/130s/moveit_resources/blob/feature-panda-moveit-perception/panda_moveit_config/launch/demo.launch.py#L18C9-L18C9),  you do not use the XML launch approach mentioned above, but add this directly into the demo python launch.

Reply:
I updated that config section to include migration from .launch to .launch.py. Hope the new change makes sense.